src/streaming/controllers/EventController.js

```diff
@@ -181,7 +205,9 @@ function EventController() {
                             activeEvents[eventId] = curr;
                         }
                         if (curr.eventStream.schemeIdUri == MPD_RELOAD_SCHEME && curr.eventStream.value == MPD_RELOAD_VALUE) {
-                            refreshManifest();
+                            if (event.duration !== 0 || event.presentationTimeDelta !== 0) { //If both are set to zero, it indicates the media is over at this point. Don't reload the manifest.
```

Comment:
@robertbryer, I think this should be:
`if (curr.duration !== 0 || curr.presentationTimeDelta !== 0)`

Reply:
Yeah, dumb mistake, thanks.